packages/stories-block/src/block/components/storiesPreview.js

```diff
@@ -75,6 +75,10 @@ function StoriesPreview(props) {
 
   const StoriesLoop = () =>
     stories.map((story) => {
+      const poster = story.meta['web_stories_poster'];
+      const featuredmedia = story._embedded?.['wp:featuredmedia']?.[0];
+
+      const posterImage = poster?.url ? poster.url : featuredmedia?.source_url;
```

Comment:
Nit-picky, but this can be simplified a bit while increasing clarity:

```suggestion
      const externalPoster = story.meta['web_stories_poster']?.url;
      const featuredImage = story._embedded?.['wp:featuredmedia']?.[0]?.source_url;

      const posterImage = externalPoster || featuredImage;
```